trino/client.py

```diff
@@ -51,21 +51,18 @@
 from time import sleep
 from typing import Any, Dict, Generic, List, Optional, Tuple, TypeVar, Union
 
-import pytz
+try:
+    from zoneinfo import ZoneInfo
+except ImportError:
+    from backports.zoneinfo import ZoneInfo
+
 import requests
-from pytz.tzinfo import BaseTzInfo
+from dateutil import tz
 from tzlocal import get_localzone_name  # type: ignore
 
 import trino.logging
 from trino import constants, exceptions
 
-try:
-    from zoneinfo import ZoneInfo  # type: ignore
-
-except ModuleNotFoundError:
-    from backports.zoneinfo import ZoneInfo  # type: ignore
```

Comment:
Previously there was no requirement in `setup.py` for the `backports.zoneinfo` package.